app/components/Quest/BadgeCard.tsx

```diff
@@ -0,0 +1,21 @@
+import moment from "moment";
+import { PlusIconBlack } from "../Icons/PlusIcon";
+
+export default function BadgeCard({name, slug, isSoldOut, saleIsOpen, saleDate}: Projects) {
```

Comment:
Why badge cards are only dealing with projects data?

Reply:
Wanted to use same structure but finally managed directly in Carousel  some badgeCard.tsx is dummy I remove